generators/client/templates/vue/src/main/webapp/app/core/jhi-navbar/jhi-navbar.component.ts.ejs

```diff
@@ -74,4 +72,8 @@ export default class JhiNavbar extends Vue {
   public hasAnyAuthority(authorities: any): boolean {
     return this.principal().hasAnyAuthority(authorities);
   }
+
+  public get activeProfiles() : string {
+    return this.$store.getters.activeProfiles;
```

Comment:
```activeProfiles``` do not need to be exposed as computed method as we do not use it directly, it seems to me that we should have here two computed methods (```swaggerEnabled``` / ```inProduction```), both using ```activeProfiles``` property from the store. 
I may be wrong but could you check database console is available in dev mode ? If not, that could be your solution.

Reply:
done